.travis.yml

```diff
@@ -1,28 +1,25 @@
+sudo: false
+dist: trusty
+
 language: php
 
 php:
-    - 5.5
     - 5.6
     - 7.0
+    - 7.1
     - hhvm
 
-env:
-    - SYMFONY_VERSION=2.3.*
-    - SYMFONY_VERSION=2.4.*
-    - SYMFONY_VERSION=2.5.*
-    - SYMFONY_VERSION=2.6.*
-    - SYMFONY_VERSION=2.7.*
-    - SYMFONY_VERSION=2.8.*
-    - SYMFONY_VERSION=3.0.*
```

Comment:
Ah yes I wanted to check on that bundle! I will update the PR to added some extra tests like in the HTTPlugBundle!